src/validations/classesValidationSchema.ts

```diff
@@ -0,0 +1,13 @@
+import { object, string, number } from 'yup';
+
+export default object({
+  body: object({
+    className: string().required().min(3, 'must be at least 3 characters long'),
+    description: string().required(),
+    trainerId: string(),
+    price: number().required(),
+    time: number(),
+    limit: number(),
+    userCount: number(),
```

Comment:
add required

Reply:
Thanks for useful comments , I hope always to give me a feedback like this .. I'm gonna fixing it